app/divisions/page.tsx

```diff
@@ -1,11 +1,20 @@
 import React from "react";
 import { Metadata } from "next";
 import DivisionsPage from "../src/components/pages/divisionsPage/divisionsPage";
+import { RepoAPICall } from "@/src/utils/api";
+import logger from "logger";
 
 export const metadata: Metadata = {
   title: "Divisions - NYPL Digital Collections",
 };
 
-export default function Divisions() {
-  return <DivisionsPage />;
+export default async function Divisions() {
+  let data;
+  try {
+    data = await RepoAPICall(`${process.env.API_URL}/api/v2/divisions`);
+  } catch (error) {
+    logger.error("Failed to fetch divisions data", error);
```

Comment:
NBD _right now_ but I'm pretty sure it will log "undefined". So eventually the proper error should be thrown in `RepoAPICall`.